notebooks/basics/quantum-circuits.ipynb

```diff
@@ -733,8 +738,8 @@
    ]
   },
   {
-   "cell_type": "raw",
-   "id": "a1320402",
+   "cell_type": "markdown",
```

Comment:
I used the raw cell type to make this part invisible — it was never quite finished and I decided not to include it. We can delete the entire cell.